web-platform/website/src/services/StudiesService.tsx

```diff
@@ -52,6 +53,20 @@ export function StudiesProvider(props: { children: React.ReactNode }) {
       (doc) => doc.data() as Study
     );
 
+    // If any attribution codes are stored in local storage, apply them to these outbound links.
+    if (typeof window !== "undefined") {
+      studies.forEach((study) => {
+        if (!study.downloadLink || !study.downloadLink.chrome || !study.downloadLink.firefox) {
+          return;
+        }
+        const chromeUrl = setAttributionCode(new URL(study.downloadLink.chrome));
```

Comment:
Take setAttribution from useAttribution object. Also window check can be placed within setAttribution.